react/src/Datasets/components/DatasetInfoDialog.tsx

```diff
@@ -104,7 +92,7 @@ export default function DatasetInfoDialog(props: DialogProp) {
                 </div>
                 <Divider />
                 <div className={classes.infoSection}>
-                    {analyses.length > 0 && (
+                    {analyses && analyses.length > 0 && (
```

Comment:
```suggestion
                    {analyses?.length > 0 && (
```

Reply:
This change doesn't work here. `Object is possibly undefined`

Reply:
That's a bit strange alright. Is this a side effect of useMemo?

Reply:
I don't think so. The same error occurs when I try `dataset?.analyses?.length` as well, avoiding useMemo.